modules/nw-metallb-configure-secondary-interface.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: PROCEDURE
+[id="nw-metallb-configure-secondary-interface_{context}"]
+= Configuring MetalLB with secondary networks
+
+From {product-title} 4.14 the default network behavior is to not allow forwarding of IP packets between network interfaces. Therefore, when MetalLB is configured on a secondary interface, you need to add a machine configuration to enable IP forwarding for only the required interfaces. 
+[NOTE]
```

Comment:
```suggestion

[NOTE]
```
Renders OK on the docs.openshift.com preview, but you'll want to check the docs.redhat.com preview after merge; hard to say if the lack of the line space breaks anything there.